compute/admin_guide/runtime_defense/runtime_defense_containers.adoc

```diff
@@ -291,11 +292,13 @@ Prisma Cloud raises alerts for each of the following cases:
 * A modified binary listens on a port,
 * A modified binary makes an outbound connection.
 
+==== Learned modles
```

Comment:
```suggestion
==== Learned models
```
@divyabhushan

Reply:
@divyabhushan however, I'm not sure this is the right title to use here, because the settings below are not only related to the learned models. Let's use something that is mentioning "Allowed" because this is the common part of the settings in this section

Reply:
@divyabhushan we can go with "Allowed activities"